### ISSUE TITLE
[Bug Report] conv test failure from discord community 

### ISSUE THREAD

**Issue by dvartaniansTT** [tenstorrent/tt-metal#14188]
**Describe the bug**
"Circular Buffer Config Error: Circular buffer size cannot be 0 B"

**To Reproduce**
`import torch
import ttnn

device_params = {"l1_small_size": 24576}
device = ttnn.CreateDevice(device_id=0, **device_params)

a = torch.rand((1, 1, 32, 32), dtype=torch.bfloat16)
b = torch.rand((1, 1, 5, 5), dtype=torch.bfloat16)
input_tensor  = ttnn.from_torch(a, layout=ttnn.TILE_LAYOUT, device=device)
weight_tensor = ttnn.from_torch(b, layout=ttnn.TILE_LAYOUT, device=device)
x = ttnn.conv2d(input_tensor=input_tensor,
                weight_tensor=weight_tensor,
                device=device,
                in_channels=1,
                out_channels=1,
                batch_size=1,
                input_height=32,
                input_width=32,
                kernel_size=[5,5],
                stride=[1,1],
                padding=[5,5],
                dilation=[1,1],
                groups=1)
res = ttnn.to_torch(x)

ttnn.close_device(device)`

**Expected behavior**
runs without the failure error

**Please complete the following environment information:**
- OS: [e.g. Ubuntu 20.04]
- Version of software: commit: 0d1eb7d95983ad7f42faadf86dc66e186f53f264

**Additional context**
this is a bug report from discord community.

**Comment by mywoodstock** [comment]
still waiting on CI pipelines -- they are having machine issues :(

**Comment by mywoodstock** [comment]
merged

**Comment by dvartaniansTT** [comment]
thanks for the prompt fix @mywoodstock.
